src/toplevel.c

```diff
@@ -420,7 +420,7 @@ static jl_module_t *call_require(jl_sym_t *var)
 // either:
 //   - sets *name and returns the module to import *name from
 //   - sets *name to NULL and returns a module to import
-static jl_module_t *eval_import_path(jl_module_t *from, jl_array_t *args, jl_sym_t **name, const char *keyword)
+static jl_module_t *eval_import_path(jl_module_t *where, jl_module_t *from, jl_array_t *args, jl_sym_t **name, const char *keyword)
```

Comment:
Due to pre-existing unfortunate naming, it is not completely unclear to me what the roles of these two modules are and are intended to be. Is `where` the module in which the statement appears? Is `from` the module from  which the bindings are being imported? The name `from` is used in many other places in this file to indicate the module in which an `import` or `using` statement appears. Or at least that was my understanding, and my attempts to change code loading appear to bear it out. It might be nice if there were comments explaining what the arguments mean. cc @vtjnash

Reply:
> Is where the module in which the statement appears? Is from the module from which the bindings are being imported?

Yes. No need to copy Jameson here; I believe this is mostly my code.

Reply:
The `from` name is used heavily in the precompilation code to mean the opposite (i.e. the module you are loading code into), I'm cc:ing him to confirm that it has the opposite meaning there.